docs/specification/unique-identifiers.md

```diff
@@ -54,6 +54,9 @@ len(`uids`) | <=           | 8
 len(`scope`)| <=           | 128, UTF-8 encoded
 len(`id`)   | <=           | 512, UTF-8 encoded
 
+Note that the 'scope' and 'id' fields can contain arbitrary characters, but cannot contain the delimiter
+sequence "::" within either of them.
+
```

Comment:
This is the behaviour within Identifier.fromKey right now:

```
key.split(Identifier.DELIMITER)
require(splitKey.length == 2, s"Invalid Identifier Key String '$key'.")
```

We could easily just split on the first instance of '::' instead of requiring that there's only one occurence. But that's how it works (when interpreting from a single string) right now.

Personally, I'd rather fix it rather than document the existing suboptimal behaviour.

Reply:
I made a Jira for this back in July: https://citrine.atlassian.net/browse/PLA-9937

This is now tagged with SELFSTART, as it's a good first ticket for someone to work on.

Closing this PR.